src/posh-git.psd1

```diff
@@ -1,10 +1,10 @@
 @{
 
 # Script module or binary module file associated with this manifest.
-ModuleToProcess = 'posh-git.psm1'
+RootModule = 'posh-git.psm1'
 
 # Version number of this module.
-ModuleVersion = '0.7.0'
+ModuleVersion = '1.0.0'
```

Comment:
'1.0.0' comes out as version 1.0.0.-1 so the order between release and pre is wrong.  :-(
We can add a field to the module manifest private data to indicate a PrereleaseVersion string that we could retrieve with `(gmo posh-git).PrivateData.PSData.PrereleaseVersion`.